src/scripts.js

```diff
@@ -35,7 +35,8 @@ Promise.all([retrieveUserData, retrieveIngredientsData, retrieveRecipesData])
     allUsersData = data[0]
     allIngredientsData = data[1];
     allRecipesData = data[2]
-    onStartup(allUsersData, allIngredientsData, allRecipesData);
+    createDomData (allUsersData, allIngredientsData, allRecipesData)
+    onStartup(domUpdates.usersDataDOM, domUpdates.ingredientsDataDOM, domUpdates.recipesDataDOM);
```

Comment:
I'm so excited this worked! Setting up the data as properties in the domUpdates object was actually pretty fun to figure out with you